src/LookingGlassXRWebGLLayer.ts

```diff
@@ -264,43 +302,48 @@ export default class LookingGlassXRWebGLLayer extends XRWebGLLayer {
 			glBindVertexArray(oldVAO)
 		}
 
-		let popup
 		window.addEventListener("unload", () => {
-			if (popup) popup.close()
-			popup = undefined
+			if (cfg.popup) cfg.popup.close()
+			cfg.popup = null
 		})
+
 		const moveCanvasToWindow = (enabled, onbeforeunload) => {
-			if (!!popup == enabled) return
+			if (!!cfg.popup == enabled) return
 
 			if (enabled) {
 				recompileProgram()
-
 				lkgCanvas.style.position = "fixed"
-				lkgCanvas.style.top = "0"
+				lkgCanvas.style.bottom = "0"
```

Comment:
this change makes the Looking Glass display show a proper hologram in most cases, instead of an incorrect view.